packages/docusaurus-theme-classic/src/theme/CodeBlock/index.tsx

```diff
@@ -112,6 +112,11 @@ export default ({
 
   const prismTheme = usePrismTheme();
 
+  // In case interleaved Markdown (e.g. when using CodeBlock as standalone component).
+  if (typeof children !== 'string') {
+    children = (children as string[]).join('');
+  }
```

Comment:
Awesome!

Yeah, I have a for of CodeBlock as a @site level component doing this. What do you think of having `Array.isArray` checking instead (since we'll do joining)? 

```js
if (Array.isArray(children))
```

And should we introduce a new local var? Not sure about the style for rewriting args.

Reply:
Agree that I'd rather have an isArray check here, as we may